### PR TITLE
FEATURE: Llama2 for summarization

### PR DESCRIPTION
**Pull request by xfalcox**
image

### CODE REVIEW

File: lib/shared/inference/hugging_face_text_generation.rb [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                        nil
+                      end
+                    end
+                  end
+              rescue IOError
+                raise if !cancelled
+              ensure
+                log.update!(
+                  raw_response_payload: response_raw,
+                  request_tokens: DiscourseAi::Tokenizer::AnthropicTokenizer.size(prompt),
```
Comment (SamSaffron) [inline lib/shared/inference/hugging_face_text_generation.rb:101]: anthropic?

**Comment by SamSaffron** [comment]
adding a chat bot after this is going to be super easy :hugs:

**Comment by SamSaffron** [comment]
Probably also worth adding instructions for devs on how to run locally, would be very cool cause more people can just hack easily on summarization or play with local bots.

**Review by romanrizzi — APPROVED** [review]
❤️

File: config/settings.yml [hunk 2]
```diff
@@ -108,6 +108,9 @@ plugins:
     choices:
       - "stable-diffusion-xl-beta-v2-2-2"
       - "stable-diffusion-v1-5"
+  ai_hugging_face_api_url:
```
Comment (romanrizzi) [inline config/settings.yml:111]: We're missing a translation for this setting.

**Pull request merged**